cxx-qt-build/src/lib.rs

```diff
@@ -93,57 +48,6 @@ fn manifest_dir() -> String {
     manifest_dir
 }
 
-/// Extract the cxx or cxx_qt module from a Rust file
-fn extract_modules(file_content: &str, rs_path: &impl AsRef<std::path::Path>) -> ExtractedModule {
```

Comment:
It seems like you inlined this method into `GeneratedCpp::new`.
Any reason for this? It makes the `new` method pretty large.

Reply:
The `extract_modules` is now effectively in `syntax::parse_qt_file`. This then responds with the enum that GeneratedCpp::new iterates through to find any CXX/CXX-Qt blocks. Before the logic of building the enum was in cxx-qt-build, now it's over in syntax of cxx-qt-gen and using syn in a more correct way. I have further changes later to make cxx-qt-gen much more modular, which then cxx-qt-build will use etc.

Reply:
Hm, okay, that is indeed cleaner.

I just think that `new` function is a bit too large now and has things in it that are at a separate level of abstraction.
Just putting that check-loop in a function `fn ensure_single_bridge` or something like that I think would clean that up a bit :)
But not a high-priority.

Reply:
Right, a separate refactor can be done in here too. And eventually we want to support multiple bridges per file ... :-)